lib/zendesk_apps_support/package.rb

```diff
@@ -212,6 +215,16 @@ def has_requirements?
       has_file?(REQUIREMENTS_FILENAME)
     end
 
+    def self.has_custom_object_requirements?(requirements_hash)
+      custom_objects = requirements_hash && requirements_hash[CUSTOM_OBJECTS_REQUIREMENT_KEY] || {}
+      types = custom_objects[CUSTOM_OBJECTS_REQUIREMENT_TYPE]
+      relationships = custom_objects[CUSTOM_OBJECTS_RELATIONSHIP_TYPE]
+
+      return true if types && types.any?
+      return true if relationships && relationships.any?
+      false
```

Comment:
nit: `(types && types.any?) || (relationships && relationships.any?)`

Reply:
Yeah, that's nicer. Thanks @fvzzy.

Reply:
Actually, this won't work. If types, or relationships are nil, this will return nil. The linter doesn't allow double negation to coerce it to a bool either. I could do a `nil?` with !, but that doesn't seem much nicer than what was previously there.

Reply:
`!!(nit: (types && types.any?) || (relationships && relationships.any?))`

Jk pls don't do this

Reply:
TIL https://stackoverflow.com/questions/12701869/nil-false-and-false-nil-in-ruby

Reply:
Linter doesn't allow the `!!` anyway, probably for the best 😆